### ISSUE TITLE
PySFML Namespacing

### ISSUE THREAD

**Issue by AphonicChaos** [Sonkun/python-sfml#88]
**TLDR: @Sonkun, do you mind clean up the sfml namespace? `import sfml as sf` would become `from sfml import sf`**

For the longest time it's been recommended to use `import sfml as sf` which makes all of PySFML's API available at the top level:

``` python
    sf.RenderWindow(sf.VideoMode(640, 480), 'Hello World')
```

Note that in this example, `sf.RenderWindow` is actually `sf.graphics.RenderWindow` and `sf.VideoMode` is actually `sf.video.VideoMode`.

My only concern here is really the polluting of the sfml namespace:

``` python
dir(sf)
['BlendMode', 'Chunk', 'CircleShape', 'Clock', 'CloseEvent', 'Color', 'Context', 'ContextSettings', 'ConvexShape', 'Drawable', 'Event', 'FocusEvent', 'Font', 'Ftp', 'FtpDirectoryResponse', 'FtpListingResponse', 
'FtpResponse', 'Glyph', 'HandledWindow', 'Http', 'HttpRequest', 'HttpResponse', 'Image', 'IpAddress', 'Joystick', 'JoystickButtonEvent', 'JoystickConnectEvent', 'JoystickMoveEvent', 'KeyEvent', 'Keyboard', 'List
ener', 'Lock', 'Mouse', 'MouseButtonEvent', 'MouseEvent', 'MouseMoveEvent', 'MouseWheelEvent', 'Music', 'Mutex', 'Pixels', 'PrimitiveType', 'Rectangle', 'RectangleShape', 'RenderStates', 'RenderTarget', 'RenderT
exture', 'RenderWindow', 'ResizeEvent', 'Shader', 'Shape', 'Socket', 'SocketDisconnected', 'SocketError', 'SocketException', 'SocketNotReady', 'SocketSelector', 'Sound', 'SoundBuffer', 'SoundBufferRecorder', 'So
undRecorder', 'SoundSource', 'SoundStream', 'Sprite', 'Style', 'TcpListener', 'TcpSocket', 'Text', 'TextEvent', 'Texture', 'Thread', 'Time', 'Transform', 'Transformable', 'TransformableDrawable', 'UdpSocket', 'V
ector2', 'Vector3', 'Vertex', 'VertexArray', 'VideoMode', 'View', 'Window', '__builtins__', '__cached__', '__doc__', '__file__', '__initializing__', '__loader__', '__name__', '__package__', '__path__', 'audio', 
'graphics', 'microseconds', 'milliseconds', 'network', 'seconds', 'sleep', 'system', 'window']
```

I'm wondering if anyone would be interested in the addition of an sf module that preserves the above behavior, but default behavior of sfml that acts as follows:

``` python
import sfml as sf

sf.graphics.RenderWindow(sf.window.VideoMode(640, 480), 'Hello World')
```

The resulting sf namespace is also cleaner:

``` python
dir(sf)
['__builtins__', '__cached__', '__doc__', '__file__', '__initializing__', '__loader__', '__name__', '__package__', '__path__', 'audio', 'graphics', 'network', 'system', 'window']
```

Then, for those wanting the old behavior:

``` python
from sfml import sf
sf.RenderWindow(sf.VideoMode(640, 480), 'Hello World')
```

In the above example, sf is it's own module, not an alias

The namespace would of course be polluted (as expected):

``` python
dir(sf)
['BlendMode', 'Chunk', 'CircleShape', 'Clock', 'CloseEvent', 'Color', 'Context', 'ContextSettings', 'ConvexShape', 'Drawable', 'Event', 'FocusEvent', 'Font', 'Ftp', 'FtpDirectoryResponse', 'FtpListingResponse', 
'FtpResponse', 'Glyph', 'HandledWindow', 'Http', 'HttpRequest', 'HttpResponse', 'Image', 'IpAddress', 'Joystick', 'JoystickButtonEvent', 'JoystickConnectEvent', 'JoystickMoveEvent', 'KeyEvent', 'Keyboard', 'List
ener', 'Lock', 'Mouse', 'MouseButtonEvent', 'MouseEvent', 'MouseMoveEvent', 'MouseWheelEvent', 'Music', 'Mutex', 'Pixels', 'PrimitiveType', 'Rectangle', 'RectangleShape', 'RenderStates', 'RenderTarget', 'RenderT
exture', 'RenderWindow', 'ResizeEvent', 'Shader', 'Shape', 'Socket', 'SocketDisconnected', 'SocketError', 'SocketException', 'SocketNotReady', 'SocketSelector', 'Sound', 'SoundBuffer', 'SoundBufferRecorder', 'So
undRecorder', 'SoundSource', 'SoundStream', 'Sprite', 'Style', 'TcpListener', 'TcpSocket', 'Text', 'TextEvent', 'Texture', 'Thread', 'Time', 'Transform', 'Transformable', 'TransformableDrawable', 'UdpSocket', 'V
ector2', 'Vector3', 'Vertex', 'VertexArray', 'VideoMode', 'View', 'Window', '__builtins__', '__cached__', '__doc__', '__file__', '__initializing__', '__loader__', '__name__', '__package__', 'microseconds', 'mill
iseconds', 'seconds', 'sleep']
```

Interesting enough, it's still smaller (94 items instead of 100) than the original implementation), not that that really matters.

PS - I was inspired by the way pyqt5 (`from pyqt5 import Qt`)
